### PR TITLE
Docs api example preface

### PR DESCRIPTION
**Pull request by jvatic**

### CODE REVIEW

File: docs/api-examples/controller.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<h1>Controller</h1>
+
+
+All requests are authenticated using [basic auth](https://en.wikipedia.org/wiki/Basic_access_authentication) where the user name is blank and the password is your controller key. You may alternatively use the `key` URL parameter with the controller key for event streams (this allows using them via JavaScript in the browser).
+
```
Comment (titanous) [inline docs/api-examples/controller.md]: This should include the `flynn -a controller env get CONTROLLER_KEY` command (I think that's the name, test it to be sure)

**Comment by jvatic** [comment]
@titanous comment addressed

File: docs/api-examples/controller.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<h1>Controller</h1>
+
+
+All requests are authenticated using [basic auth](https://en.wikipedia.org/wiki/Basic_access_authentication) where the user name is blank and the password is your controller key. You may alternatively use the `key` URL parameter with the controller key for event streams (this allows using them via JavaScript in the browser). You can retreive your controller key by running `flynn -a controller env get AUTH_KEY`.
```
Comment (titanous) [inline docs/api-examples/controller.md]: s/retreive/retrieve

s/user name/username

Comment (jvatic) [inline docs/api-examples/controller.md]: Fixed

File: docs/api-examples/controller.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<h1>Controller</h1>
+
+
+All requests are authenticated using [basic auth](https://en.wikipedia.org/wiki/Basic_access_authentication) where the username is blank and the password is your controller key. You may alternatively use the `key` URL parameter with the controller key for event streams (this allows using them via JavaScript in the browser). You can retrieve your controller key by running `flynn -a controller env get AUTH_KEY`.
+
+All paths below are relative to `controller.CLUSTER_DOMAIN`.
```
Comment (titanous) [inline docs/api-examples/controller.md]: This line needs to be clearer, something like "The API root URL is the controller domain, by default `https://controller.$CLUSTER_DOMAIN`."

**Comment by titanous** [comment]
ping

**Comment by jvatic** [comment]
updated

**Comment by titanous** [comment]
LGTM

**Pull request merged**